site.yml

```diff
@@ -159,6 +159,30 @@
       tags: ['openstack', 'nova', 'data']
       when: not ironic.enabled|bool
 
+- name: ceph monitors
+  gather_facts: force
+  hosts: ceph_monitors
+  roles:
+    - role: ceph-common
+      tags: ['ceph', 'ceph-common']
+      when: ceph.enabled|bool
```

Comment:
Also the conditional here is unnecessary. You wouldn't put a host in the ceph_monitors group if you weren't enabling ceph. Same for all the conditionals in these group specific plays.